kotlin/src/test/kotlin/code/retreat/GreetingTest.kt

```diff
@@ -0,0 +1,15 @@
+package code.retreat
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class GreetingTest {
+
+    val greeting = Greeting()
+
+    @Test
```

Comment:
Should we have a common style for the tests?
If I look at the Java part I can see
```
class AppTest {
    @Test void appHasAGreeting() {
        App classUnderTest = new App();
        assertNotNull(classUnderTest.getGreeting(), "app should have a greeting");
    }
}
```
The name of the class is AppTest and the name of the test is appHasAGreeting instead of AppTest and hasAGreeting.
That being said not all the language use classes to embed tests so it might be difficult to be consistent across the board.

Reply:
It's even worse if you look at the python and typescript starters , they have entirely different names and structures. I don't think it matters though, the goal is to provide boiler plate code.